### PR TITLE
Software 0.30.2 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://github.com/astronomer/astronomer/releases/tag/v0.30.2?notification_referrer_id=NT_kwDOBHHpmbM0NDU4NDQxMjEwOjc0NTc0MjMz

### CODE REVIEW

File: software/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,19 @@ description: Astronomer Software release notes.
 
 0.30 is the latest stable and long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
 
+## 0.30.2
+
+Release date: September 23, 2022
+
+### Additional improvements
+
+- You can now set a `securityContext` for Fluentd Pods and containers using the [Fluentd Helm chart](https://github.com/astronomer/astronomer/blob/master/charts/fluentd/values.yaml).
```
Comment (kwcanuck) [inline software/release-notes.md:18]:
```suggestion
- You can now use the [Fluentd Helm chart](https://github.com/astronomer/astronomer/blob/master/charts/fluentd/values.yaml) to set a `securityContext` for Fluentd Pods and containers.
```

File: software/release-notes.md [hunk 2]
```diff
@@ -9,6 +9,19 @@ description: Astronomer Software release notes.
...
+- Improved the startup time for the platform NATS server.
+
+### Bug fixes
+
+- Fixed several CVEs as a result of updating images for system components.
```
Comment (kwcanuck) [inline software/release-notes.md:23]: I assume that you don't want to define _CVE_? Typically, you define acronyms on first use.

**Review by kwcanuck — APPROVED** [review]

File: software/release-notes.md [hunk 3]
```diff
@@ -9,6 +9,19 @@ description: Astronomer Software release notes.
 
 0.30 is the latest stable and long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).
 
+## 0.30.2
+
+Release date: September 23, 2022
```
Comment (vishwas-astro) [inline software/release-notes.md:14]: was released on Sep 22

File: software/release-notes.md [hunk 4]
```diff
@@ -9,6 +9,19 @@ description: Astronomer Software release notes.
...
+
+### Additional improvements
+
+- You can now use the [Fluentd Helm chart](https://github.com/astronomer/astronomer/blob/master/charts/fluentd/values.yaml) to set a `securityContext` for Fluentd Pods and containers.
+- Improved the startup time for the platform NATS server.
+
```
Comment (vishwas-astro) [inline software/release-notes.md:20]: The main bug fix that went in to this release was https://github.com/astronomer/issues/issues/5046. Dont we want to add a note about that? cc: @cmarteepants

File: software/release-notes.md (hunk 3 above)
Comment (jwitz) [inline software/release-notes.md:14]:
```suggestion
Release date: September 22, 2022
```

File: software/release-notes.md (hunk 4 above)
Comment (jwitz) [inline software/release-notes.md:20]:
```suggestion
- You can now configure external containers in the `astronomer.houston.config` section of the Astronomer Helm chart.
```

File: software/release-notes.md [hunk 5]
```diff
@@ -9,6 +9,19 @@ description: Astronomer Software release notes.
...
+## 0.30.2
+
+Release date: September 22, 2022
+
+### Additional improvements
+
+- You can now use the [Fluentd Helm chart](https://github.com/astronomer/astronomer/blob/master/charts/fluentd/values.yaml) to set a `securityContext` for Fluentd Pods and containers.
+- Improved the startup time for the platform NATS server.
+- You can now configure external containers in the `astronomer.houston.config` section of the Astronomer Helm chart.
+### Bug fixes
```
Comment (jwitz) [inline software/release-notes.md:21]:
```suggestion

### Bug fixes
```

**Pull request merged**